sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -463,7 +505,6 @@ export class EventHubSender extends LinkEntity {
    */
   private _trySendBatch(
     message: AmqpMessage | Buffer,
-    tag: any,
```

Comment:
I'm not familiar with the `tag` concept. Is this something that Service Bus supports, but Event Hubs doesn't?

Reply:
We are setting `tag` as undefined in the Service Bus as well. As per my understanding, rhea removes the tag length from the connection max frame size and set the tag in the` tag` property before sending.

More more details refer: https://github.com/amqp/rhea/blob/master/lib/session.js#L145